system/Session/Session.php

```diff
@@ -268,7 +268,7 @@ public function start()
      *
      * @deprecated Use destroy() instead.
      */
-    public function stop()
+    public function stop(): void
```

Comment:
this is bc break on non-final class, see https://3v4l.org/LSYES , you may need to use `@return void` instead for safer